tests/test_benchmark.py

```diff
@@ -178,6 +184,13 @@ def __init__(self, name, engine, extra_args=[], env={}, binaryen_opts=[]):
       self.env[k] = v
     self.binaryen_opts = binaryen_opts[:]
 
+  def prepare(self):
+    # add the feature flags to build commands for libraries as well (easier
+    # this way than editing each one separately). this isn't needed for
+    # native builds, so the comparison is still apples-to-apples.
+    assert not os.environ.get('EMMAKEN_CFLAGS')
+    os.environ['EMMAKEN_CFLAGS'] = ' '.join(LLVM_FEATURE_FLAGS)
```

Comment:
Eww.. Not a fan of this way of injecting CFLAGS.   I was really hoping to remove EMMAKEN_CFLAGS at some point :(

Ant no just do this __init__?

Reply:
It is a useful hack for build systems you don't want to take the time to fully understand :wink:  I get that it's ugly, but it saves so much time...

We can't do this in init, as init is called when we create the instance of the Benchmarker, and we create those during startup of the entire test harness (this file is loaded before we parse the args to know if you are even running a benchmark).

Reply:
Hmm, I still don't really like this.   You have a CFLAGS in the Makefile that can be used for this purpose.

I think this can be done by add 'CFLAGS' to the env_init argument to lib_builder on line 187 in `EmscriptenBenchmarker:build`.